src/Testing/src/xunit/SkipOnHelixAttribute.cs

```diff
@@ -52,7 +52,7 @@ private bool ShouldSkip()
 
             var targetQueue = GetTargetHelixQueue().ToLowerInvariant();
 
-            if (Queues.Contains("All.OSX") && targetQueue.StartsWith("osx"))
+            if (Queues.Contains("All.OSX") && targetQueue.StartsWith("osx", StringComparison.Ordinal))
```

Comment:
@HaoK Should this be `Ordinal`, or `OrdianalIgnoreCase` do you think?

Reply:
OrdinalIgnoreCase seems more appropriate, since we don't actually care about casing differences

Reply:
```suggestion
            if (Queues.Contains("All.OSX") && targetQueue.StartsWith("osx", StringComparison.OrdinalIgnoreCase))
```